### PR TITLE
Feat/get user by Id

### LINKED ISSUE #118: Get user by Id

### PR DESCRIPTION
**Pull request by vzart**

### CODE REVIEW

**Review by StarryFolf — APPROVED** [review]
LGTM

File: src/Application/Users/Queries/GetUserById.cs [hunk 1]
```diff
@@ -8,4 +11,29 @@ public record Query : IRequest<UserDto>
...
+        }
+
+        public async Task<UserDto> Handle(Query request, CancellationToken cancellationToken)
+        {
+            var user = await _context.Users
+                .FirstOrDefaultAsync(x => x.Id.Equals(request.UserId), cancellationToken: cancellationToken);
+
+            if (user is null)
+            {
+                throw new KeyNotFoundException("User does not exist");
```
Comment (kaitoz11) [inline src/Application/Users/Queries/GetUserById.cs]: "User does not exist."

File: tests/Application.Tests.Integration/Users/Queries/GetUserByIdTests.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        {
+            UserId = Guid.NewGuid(),
+        };
+        
+        // Act
+        var action = async () => await SendAsync(query);
+        
+        // Assert
+        await action.Should().ThrowAsync<KeyNotFoundException>()
+            .WithMessage("User does not exist");
```
Comment (kaitoz11) [inline tests/Application.Tests.Integration/Users/Queries/GetUserByIdTests.cs]: "User does not exist."

**Review by kaitoz11 — CHANGES_REQUESTED** [review]
excuse me!

**Review by kaitoz11 — APPROVED** [review]
LGTM

**Pull request merged**